pmm-app/src/pmm-add-instance-app-panel/AddInstance/AddRemoteInstance/AddRemoteInstance.tsx

```diff
@@ -186,10 +186,10 @@ const AddRemoteInstance = props => {
               <span className="description">Port your service is listening on</span>
             </div>
             <div className="add-instance-panel">
-              <InputField form={form} name="username" placeholder="*Username" required={true} />
+              <InputField form={form} name="username" placeholder="Username" required={true} />
               <span className="description">Your database user name</span>
 
-              <PasswordField form={form} name="password" placeholder="*Password" required={true} />
+              <PasswordField form={form} name="password" placeholder="Password" required={true} />
```

Comment:
Username and Password fields are required as far as I know, why are we removing the * asterisk? or I am getting it wrong?

Reply:
I just changed functionality a little, now Input field component adds atersik if required flag is used